src/ocamlorg_frontend/pages/community.eml

```diff
@@ -36,6 +36,10 @@ Layout.render
           <div class="text-base font-semibold"><img src="/img/community/discourse.svg" alt="" class="h-8 w-8 mr-2 inline-block"> Discuss</div>
           <div class="font-normal text-xs mt-4">Discuss with the community, ask questions and share your work on OCaml.</div>
         </a>
+        <a href="https://watch.ocaml.org/" class="border-gray-200 border p-8 rounded-xl card-hover">
+          <div class="text-base font-semibold"><img src="/img/community/peertube.svg" alt="" class="h-8 w-8 mr-2 inline-block"> Watch</div>
+          <div class="font-normal text-xs mt-4">Watch community videos like past OCaml Workshop events.</div>
+        </a>
         <a href="https://github.com/ocaml/ocaml" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/github.svg" alt="" class="h-8 w-8 mr-2 inline-block"> Github</div>
```

Comment:
```suggestion
          <div class="text-base font-semibold"><img src="/img/community/github.svg" alt="" class="h-8 w-8 mr-2 inline-block"> GitHub</div>
```

unrelated but noticed this typo...

Reply:
good catch -- this is the first time I've realised the H is supposed to capitalised too!